<launch>
    <node pkg="airsim_based_object_recognition" name="airsim_based_object_recognition_node" type="airsim_based_object_recognition_node.py" output="screen">
        <param name="mesh_match_regex" value="OrangeBall" />
        <param name="camera_name_list" value="[front_center_custom, front_left_custom, front_right_custom]" />
    </node>
</launch>
